Makefile

```diff
@@ -126,9 +126,15 @@ helm-index:
 			     $(MAKE) semaphore-run-workflow
 
 # Creates the tar file used for installing Calico on OpenShift.
-bin/ocp.tgz: manifests/ocp/
-	mkdir -p bin
-	tar czvf $@ -C manifests/ ocp
+bin/ocp.tgz: manifests/ocp/ bin/yq
+	mkdir -p bin/tmp
+	cp -r manifests/ocp bin/tmp/
+	$(DOCKER_RUN) $(CALICO_BUILD) /bin/bash -c "                                        \
+		for file in bin/tmp/ocp/*crd* ;                                                 \
+        	do bin/yq -i 'del(.. | select(has(\"description\")).description)' \$$file ; \
```

Comment:
This may create a problem if we start to rely on fields that are legitimately called "description" . There are two in the repo: OpsGenieConfig and PagerDutyConfig, neither of which we use today.